docs/content/deployment/guides/kubernetes/customizing-your-deployment.mdx

```diff
@@ -7,9 +7,50 @@ description: This section covers common ways to customize your Dagster Helm depl
 
 This section covers common ways to customize your Dagster Helm deployment.
 
-## Job or Op Kubernetes Configuration
+## Specifying custom Kubernetes configuration
+
+Dagster allows you to pass custom configuration through to the Kubernetes Jobs and Pods created by Dagster during execution.
```

Comment:
```suggestion
Dagster allows you to pass custom configuration to the Kubernetes Jobs and Pods created by Dagster during execution.
```

I took `through` out because it didn't feel necessary, but feel free to add it back if it does convey some special meaning